suite-common/wallet-utils/src/transactionUtils.ts

```diff
@@ -46,30 +46,47 @@ export const parseDateKey = (key: string) => {
     return d;
 };
 
+/* Convert date to string in YYYY-MM format */
+export const getMonthKey = (d: Date) => `${d.getFullYear()}-${d.getMonth() + 1}`;
+
+/** Parse Date object from a string in YYYY-MM format */
+export const parseMonthKey = (key: string) => {
+    const parts = key.split('-');
+    const d = new Date(Number(parts[0]), Number(parts[1]) - 1);
+    return d;
+};
+
 /**
  * Returns object with transactions grouped by a date. Key is a string in YYYY-MM-DD format.
  * Pending txs are assigned to key 'pending'.
  *
  * @param {WalletAccountTransaction[]} transactions
  */
-export const groupTransactionsByDate = (transactions: WalletAccountTransaction[]) =>
+export const groupTransactionsByDate = (
+    transactions: WalletAccountTransaction[],
+    groupSize: 'day' | 'month' = 'day',
```

Comment:
I think `groupBy` is more standartized name.

Reply:
fixed [30b5f1a](https://github.com/trezor/trezor-suite/pull/7392/commits/30b5f1a27811ed560da7091928ce739d439d0ca9)